examples/expressApp/package-lock.json

```diff
@@ -1,1947 +1,8 @@
 {
```

Comment:
No need to delete this file, from what I've read, for a final project (not the library), like this example server app, it is recommended to include this lock file. To make sure dependency is locked & user can always get same result with locked dependency.